lib/components/mobile/batch-results-screen.js

```diff
@@ -40,30 +40,42 @@ const ExpandMapButton = styled(Button)`
   z-index: 999999;
 `
 
-const VIEW = uiActions.ItineraryView
-
 const NARRATIVE_SPLIT_TOP_PERCENT = 45
 
+// Styles for the results map also include prop-independent styles copied from mobile.css.
+const ResultsMap = styled.div`
+  bottom: ${props => props.expanded ? '0' : `${100 - NARRATIVE_SPLIT_TOP_PERCENT}%`};
+  display: ${props => props.visible ? 'inherit' : 'none'};
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 100px;
+`
+
+const StyledResultsError = styled(ResultsError)`
+  display: ${props => props.visible ? 'inherit' : 'none'};
+  top: ${props => props.visible ? (props.expanded ? '100px' : `${NARRATIVE_SPLIT_TOP_PERCENT}%`) : '100%'};
+  transition: top 300ms;
+`
+
+const VIEW = uiActions.ItineraryView
```

Comment:
On second look, I find the re-assignment/renaming of ItineraryView a bit confusing. It feels like a bit of an unnecessary redirection for the reader. Recommendation: `const {ItineraryView} = uiActions`

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/48bafb663209bf2e84d22b0c491709cba7c31919.